### PR TITLE
[device/centec] Replace os.system and remove subprocess with shell=True

### PR DESCRIPTION
**Pull request by maipbui**
Signed-off-by: maipbui <maibui@microsoft.com>

#### Why I did it
`subprocess.Popen()` and `subprocess.run()` is used with `shell=True`, which is very dangerous for shell injection.
`os` - not secure against maliciously constructed input and dangerous if used to evaluate dynamic content
#### How I did it
Replace `os` by `subprocess`, remove `shell=True`
Remove unused functions
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [X] 201911
- [X] 202006
- [X] 202012
- [X] 202106
- [X] 202111
- [X] 202205

### CODE REVIEW

File: device/centec/arm64-centec_e530_24x2c-r0/platform_reboot [hunk 1]
```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/python
-import os
 
 def main():
     # reboot the system
-    os.system('echo 502 > /sys/class/gpio/export')
-    os.system('echo out > /sys/class/gpio/gpio502/direction')
-    os.system('echo 1 > /sys/class/gpio/gpio502/value')
+    with open('/sys/class/gpio/export', 'w') as file:
+        file.write('502')
```
Comment (qiluo-msft) [inline device/centec/arm64-centec_e530_24x2c-r0/platform_reboot:6]: >502 [](http://example.com/codeflow?start=20&length=3)

Do you miss the trailing new line?

Comment (maipbui) [reply device/centec/arm64-centec_e530_24x2c-r0/platform_reboot]: I added `\n`, thank you.

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
@LuiSzee @guxianghong @taochengyi @yangbashuang 
Could you help review and verify?

**Comment by guxianghong** [comment]
> @LuiSzee @guxianghong @taochengyi @yangbashuang Could you help review and verify?

Ok, we will verify it.

**Comment by LuiSzee** [comment]
subprocess.run not work on older version SONiC image, such as 201911

```
Python 2.7.17 (default, Jul  1 2022, 15:56:32)
[GCC 7.5.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocess
>>> subprocess.run('sleep', '1')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'run'
```

**Comment by maipbui** [comment]
> subprocess.run not work on older version SONiC image, such as 201911
> 
> ```
> Python 2.7.17 (default, Jul  1 2022, 15:56:32)
> [GCC 7.5.0] on linux2
> Type "help", "copyright", "credits" or "license" for more information.
> >>> import subprocess
> >>> subprocess.run('sleep', '1')
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
> AttributeError: 'module' object has no attribute 'run'
> ```

@LuiSzee I changed it to `subprocess.call()`, thanks for your feedback.

**Comment by maipbui** [comment]
@LuiSzee @guxianghong @taochengyi @yangbashuang Do you have any further comments?

**Comment by LuiSzee** [comment]
@maipbui I have verified it on centec platform.

**Pull request merged**